hack/update-codegen.sh

```diff
@@ -71,4 +73,21 @@ ${clientgen} --clientset-name=federation_internalclientset --clientset-path=k8s.
 ${clientgen} --clientset-name=federation_release_1_5 --clientset-path=k8s.io/kubernetes/federation/client/clientset_generated --input="../../federation/apis/federation/v1beta1","api/v1","extensions/v1beta1" --included-types-overrides="api/v1/Service,api/v1/Namespace,extensions/v1beta1/ReplicaSet,api/v1/Secret,extensions/v1beta1/Ingress,extensions/v1beta1/Deployment,extensions/v1beta1/DaemonSet,api/v1/ConfigMap,api/v1/Event"   "$@"
 ${setgen} "$@"
 
+LISTERGEN_APIS=(
```

Comment:
Especially if we have each GV for listers in its own package, this won't be an issue.